docs/ops.html

```diff
@@ -866,7 +866,7 @@ <h4><a id="remote_jmx" href="#remote_jmx">Security Considerations for Remote Mon
         <td></td>
       </tr>
       <tr>
-        <td># of under replicated partitions (|ISR| &lt |all replicas|)</td>
+        <td># of under replicated partitions (|non-reassigning replicas| - |ISR| &gt 0)</td>
```

Comment:
Should we reword this to "the number of non-reassigning replicas - the number of ISR" to make it explicit that something like
```
r=[1,2,3,4,5,6]
target_replicas=[4,5,6]
isr=[3,4,5]
```
Would not count as URP?

Not sure if people may read the above exaple as `[1,2,3] - [3,4,5] > 0` given the current wording

Reply:
Cool, I'll reword that.